tensorzero-core/src/evaluations/mod.rs

```diff
@@ -92,6 +95,15 @@ pub struct LLMJudgeConfig {
     pub cutoff: Option<f32>,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+#[cfg_attr(test, derive(ts_rs::TS))]
+#[cfg_attr(test, ts(export))]
+pub struct RegexConfig {
+    pub regex: String,
+    #[serde(default)]
+    pub cutoff: Option<f32>,
```

Comment:
as far as I can tell this doesn't get used? just return None in the place this gets called and remove